cli_li3ds/import_orimatis.py

```diff
@@ -283,7 +283,7 @@ def datasource_image(session, referential, datasource,
     if image_file_ext:
         image_path = image_path.with_suffix(image_file_ext)
     image_path = os.path.normpath(str(image_path))
-    uri = 'file:{}'.format(image_path)
+    uri = 'file://{}'.format(image_path)
```

Comment:
Ok, it wasn't clear for me since the `// `is commonly used in uri schemes.

`//` is known as authority as explained here https://tools.ietf.org/html/rfc3986#section-3, we dont' need this for now.

The current `check_datasource_uri` doesn't check the _`path` part.
I think we could move the check_datasource_uri to pl/python and use the `urlparse` module for that. It will be easier to check the file existence (and client won't have to worried about giving 1 or 3 slashes since `urlparse('file:///home/ludo') == urlparse('file:/home/ludo')`)